nucliadb/nucliadb/writer/api/v1/services.py

```diff
@@ -28,19 +28,28 @@
     DelLabelsRequest,
     DelVectorSetRequest,
     DetWidgetsRequest,
+    NewEntitiesGroupRequest,
+    NewEntitiesGroupResponse,
     OpStatusWriter,
     SetEntitiesRequest,
     SetLabelsRequest,
     SetSynonymsRequest,
     SetWidgetsRequest,
+    UpdateEntitiesGroupRequest,
+    UpdateEntitiesGroupResponse,
 )
 from starlette.requests import Request
 
 from nucliadb.writer.api.v1.router import KB_PREFIX, api
 from nucliadb.writer.resource.vectors import create_vectorset  # type: ignore
-from nucliadb_models.entities import EntitiesGroup
+from nucliadb_models.entities import (
+    CreateEntitiesGroupPayload,
+    EntitiesGroup,
+    UpdateEntitiesGroupPayload,
+)
 from nucliadb_models.labels import LabelSet
 from nucliadb_models.resource import NucliaDBRoles
+from nucliadb_models.responses import HTTPConflict, HTTPNotFound
```

Comment:
Should these just be in the `nucliadb` package? `nucliadb_models` is more for communicating types to external parties but we probably don't want or need it to have a `fastapi` dependency right?

Reply:
Moved to `nucliadb/models`